OracleApplicationExpress/README.md

```diff
@@ -0,0 +1,123 @@
+# Oracle REST Data Services on Docker
+
+Sample Docker build files to facilitate installation, configuration, and environment setup for DevOps users.
+For more information about Oracle REST Data Services (ORDS) please see the [ORDS Documentation](http://www.oracle.com/technetwork/developer-tools/rest-data-services/documentation/index.html).
+
+## How to build and run
+
+This project offers sample Dockerfiles for Oracle REST Data Services
+
+To assist in building the images, you can use the [buildContainerImage.sh](dockerfiles/buildContainerImage.sh) script. See below for instructions and usage.
+
+The `buildContainerImage.sh` script is just a utility shell script that performs MD5 checks and is an easy way for beginners to get started. Expert users are welcome to directly call `docker build` with their preferred set of parameters.
+
+### Building Oracle REST Data Services Install Images
+
+**IMPORTANT:** You can provide the installation binaries of ORDS and put them into the `dockerfiles` folder. You only need to provide the binaries for the version you are going to install.\
+The binaries can be downloaded from the [Oracle Technology Network](http://www.oracle.com/technetwork/developer-tools/rest-data-services/downloads/index.html) and placed in `OracleRestDataServices/dockerfiles` directory. Note that you must not uncompress the binaries. The script will handle that for you and fail if you uncompress them manually!  
+
+**If no binaries are provided, [the latest ords zip](https://download.oracle.com/otn_software/java/ords/ords-latest.zip) file is downloaded automatically.**
+
+The image builds on top of the `oracle/serverjre:8` image which is also provided in this repository, see [OracleJava](../OracleJava). This base image is fetched from the container registry. So for successful fetch the user needs to login to the [container-registry](container-registry.oracle.com) and accept the license agreement.\
+The user can also build the `oracle/serverjre:8` image locally before building this image using the [OracleJava](../OracleJava) repo. After building this image locally, the user can run the following command:
+
+```bash
+./buildContainerImage.sh -o '--build-arg BASE_IMAGE=<local-image>'
+```
+
+Before you build the image make sure that you have provided the installation binaries and put them into the right folder. Once you have done that go into the **dockerfiles** folder and run the **buildContainerImage.sh** script:
+
+```bash
+    [oracle@localhost dockerfiles]$ ./buildContainerImage.sh -h
+    
+    Usage: buildContainerImage.sh [-i] [-o] [Docker build option]
+    Builds a Docker Image for Oracle Rest Data Services
+    
+    Parameters:
+       -i: ignores the MD5 checksums
+       -o: passes on Docker build option
+    
+    LICENSE UPL 1.0
+    
+    Copyright (c) 2014-2017 Oracle and/or its affiliates. All rights reserved.
```

Comment:
This is a brand new file. It should be `Copyright (c) 2025 Oracle and/or its affiliates.`

Oracle no longer includes the phrase "all rights reserved"